libraries/apollo-mockserver/src/commonMain/kotlin/com/apollographql/apollo3/mockserver/MockServer.kt

```diff
@@ -92,19 +92,23 @@ interface MockServer : Closeable {
       this.handlePings = handlePings
     }
 
-    fun tcpServer(tcpServer: TcpServer, port: Int = 0) = apply {
+    fun tcpServer(tcpServer: TcpServer) = apply {
```

Comment:
Moved `port` to a separate function as `tcpServer` has a port configured already.